codalab/requirements/common.txt

```diff
@@ -61,6 +60,8 @@ gunicorn==19.7.0
 git+https://github.com/Tthomas63/django-switchuser.git@eb8b71622201f66ef1862a4976c12ef83145b5e8
 Sphinx==1.4.1
 
+#django-allauth==0.11.1
+git+https://github.com/gibsonbailey/django-allauth.git@codalab
```

Comment:
Could we add a comment as to why we did this? Like, `# This fork patches potential ?next= phishing attack`